service/src/main/java/com/ge/predix/acs/privilege/management/ResourcePrivilegeManagementController.java

```diff
@@ -133,15 +127,13 @@ private void failIfParentsSpecified(final List<BaseResource> resources) {
         }
 
         if (resource == null) {
-            return notFound();
```

Comment:
Not sure if returning an error is the best approach here.  May be returning an empty body alone with Not Found status is better?